Assets/PlayroomKit/PlayroomKit.cs

```diff
@@ -971,6 +971,167 @@ public class Dpad
             public string y;
         }
 
+
+        // RPC:
+        public enum RPCModes
+        {
+            ALL,
+            OTHERS,
+            HOST
+        }
+
+        private static Action<string, string> RpcRegisterCallback = null;
+
+        [DllImport("__Internal")]
+        private extern static void RpcRegisterInternal(string name, Action<string, string> rpcRegisterCallback, string onResponseReturn = null);
+
+        public static void RpcRegister(string name, Action<string, string> rpcRegisterCallback, string onResponseReturn = null)
+        {
+            RpcRegisterCallback = rpcRegisterCallback;
+            RpcRegisterInternal(name, InvokeRpcRegisterCallBack, onResponseReturn);
+        }
+
+        [MonoPInvokeCallback(typeof(Action<string, string>))]
+        private static void InvokeRpcRegisterCallBack(string dataJson, string senderJson)
+        {
+            try
+            {
+                if (!Players.ContainsKey(senderJson))
+                {
+                    var player = new Player(senderJson);
+                    Players.Add(senderJson, player);
+                }
+                else
+                {
+                    Debug.LogWarning($"Players dictionary already has a player with ID: {senderJson}!");
+                }
+            }
+            catch (Exception ex)
+            {
+                Debug.LogError(ex.Message);
+            }
+            RpcRegisterCallback?.Invoke(dataJson, senderJson);
+        }
+
+        [DllImport("__Internal")]
+        private extern static void RpcCallInternal(string name, string data, RPCModes mode, Action callbackOnResponse);
+
+        private static Dictionary<string, List<Action>> OnResponseCallbacks = new Dictionary<string, List<Action>>();
+        private static List<string> RpcEventNames = new List<string>();
+
+        public static void RpcCall(string name, object data, RPCModes mode, Action callbackOnResponse)
```

Comment:
In PlayroomKit JS, the function name is RPC.call, which implies RPC is an object (or a Class!) containing `call` as a function. Why didn't we follow that in particular?

Reply:
(An explanation would do if this feature is not needed right now)

Reply:
Makes sense for now. If we expand this later on, we may need to put it in a class, which would be a breaking change.